js/page-specific/baltcogo-app/BaltCoGoReporterCtrl.js

```diff
@@ -288,6 +279,25 @@
 			mapServiceComposite.createMarker(map, latitude, longitude);
 		};
 
+		function hasProperty(obj, prop) {
+			if (!obj) { return false; }
+			return Object.prototype.hasOwnProperty.call(obj, prop);
+		}
+
+		self.shouldRequireLocation = function shouldRequireLocation() {
+			var categoryHasRequiresLocationProperty = hasProperty(self.category, 'requiresLocation');
+			var subCategoryyHasRequiresLocationProperty = hasProperty(self.subCategory, 'requiresLocation');
```

Comment:
Should `'requiresLocation'` be provided as a constant?

Reply:
@ascire done

Reply:
Neat.  Don't forget rebuild.

Reply:
@ascire thanks, rebuild...definitely forgot